### PR TITLE
Delay expansion for assignments

### PR DESCRIPTION
**Pull request by skaex**
This is an update on getting the assignment expansion to work properly.

For more context see this Asana task: https://app.asana.com/0/1172786304330247/1187318886285193

### CODE REVIEW

File: app/views/assignments/show.html.erb [hunk 1]
```diff
@@ -252,6 +252,6 @@
   <% if @assignment.new_html? %>
     window.BRAVEN_NEW_HTML = true;
   <% end %>
-
-  bzRetainedInfoSetup();
+    bzRetainedInfoSetup();
+    window.addEventListener('load', function() { expandCohortMagicFields(); })
```
Comment (skaex) [inline app/views/assignments/show.html.erb:256]: This calls the expansion logic when the window loads.

File: app/views/layouts/_foot.html.erb [hunk 2]
```diff
@@ -145,4 +145,7 @@
 </div>
 
 
+<script type="application/json" id="my-current-cohort-json"><%= current_cohort.to_json.gsub("/script", "").html_safe %></script>
```
Comment (skaex) [inline app/views/layouts/_foot.html.erb:148]: This needed to be hear because the `bzRetainedInfoSetup` actually needs `ENV` loaded first

**Comment by skaex** [comment]
This seems to work now (locally).

File: app/views/layouts/_foot.html.erb (hunk 2 above)
Comment (sadleb) [reply app/views/layouts/_foot.html.erb:148]: Ahh, kk. This fix makes more sense to me than the last attempt. Looks good!

**Pull request merged**